classes/target/target_sftp_key.php

```diff
@@ -198,7 +198,7 @@ public function is_available() {
         }
 
         if (!$this->logginresult) {
-            $this->log('login', 'Login failed using ' . $this->settings['username'], logger::TYPE_ERROR);
+            $this->log('login', get_string('login_failed_target_sftp_email', 'tool_etl'), logger::TYPE_ERROR);
```

Comment:
This no longer shows $this->settings['username']

lang strings can have arguments, eg

https://docs.moodle.org/dev/String_API#Run-time_parameters